packages/readable-js-assets-webpack-plugin/test/__snapshots__/build.js.snap

```diff
@@ -16,7 +16,7 @@ notMinified();
 exports[`ReadableJsAssetsWebpackPlugin should produce the expected output: Asset file index.min.js should match snapshot 1`] = `"console.log("hello");"`;
 
 exports[`ReadableJsAssetsWebpackPlugin should produce the expected output: Asset file view.js should match snapshot 1`] = `
-"/******/ (() => { // webpackBootstrap
+"/******/ (function() { // webpackBootstrap
```

Comment:
Weird, I though arrow support was good enough :P

Reply:
There might be a reason why webpack wraps code with a regular function now 🤷🏻